pkg/apiserver/handlers/webhook/convert_crd_test.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webhook
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var (
+	unstructuredTestData = map[string]interface{}{
+		"kind":       "kind",
+		"apiVersion": "apiVersion",
+	}
+	invalidUnstructuredTestData = map[string]interface{}{
+		"test": "test",
+	}
```

Comment:
I don't think that "unstructured" needs to be in the name of these variables. It's a bit confusing because these variables are not of type `unstructured.Unstructured`. So maybe just `testData` and `invalidTestData`.

Reply:
Renamed.